addons/dark-www/experimental_scratchr2.css

```diff
@@ -24,10 +24,20 @@ label,
 #topnav ul.account-nav .logged-in-user .dropdown-menu,
 #topnav li.logout.divider input,
 #topnav ul.account-nav .sign-in .popover {
-  background-color: var(--darkWww-navbar);
+  background-color: var(--darkWww-navbar-scratchr2);
 }
-#topnav ul.account-nav .sign-in .popover .arrow:after {
-  border-bottom-color: var(--darkWww-navbar);
+#topnav ul.site-nav li,
+#topnav ul.site-nav li.last,
+#topnav ul.account-nav > li,
+#topnav ul.account-nav > li.join-scratch,
+#topnav ul.account-nav.logged-in > li,
+#topnav ul.account-nav ul.user-nav li.logout.divider,
+#topnav ul.account-nav .sign-in .popover {
+  border-color: var(--darkWww-navbar-scratchr2);
```

Comment:
A minor thing - when the navigation bar is set to a transparent color, the borders become visible.

![A transparent purple navigation bar with visible borders between each link](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/4df0ae23-62d0-4493-9d77-3558e01a0929)

Reply:
Fixed.